src/lsst/cm/tools/core/error_code_decisions.yaml

```diff
@@ -253,6 +328,17 @@ pandaErrorCode:
             rescue: 0
             panda: 0
             intensity: 0.001
+        psfex:
```

Comment:
Same as `p_psfex`?  

I'm confused; are duplicate items below intentional?

Reply:
Currently we have different entries for errors which come up under `pilot, 1305` and `trans, 1` -- We need the pilot errors for sure, as the `trans, 1` errors always only read `Unspecified error, consult log file`. Many of the errors that I marked as `p_` in the `trans, 1` section were initially not promoted to pilot. I'm not sure that they won't be in future, so I usually try to add an entry for `pilot, 1305` as well as the ones that will show up in the logs. Since there are more of these I think the best solution might be to find a way to note these as the same error